apps/address-service/domains/address/schema/AddressSource.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Generated by `createschema address.AddressSource 'source:Text;'`
+ */
+
+const { Text, Relationship } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const access = require('@address-service/domains/address/access/AddressSource')
+
+
+const AddressSource = new GQLListSchema('AddressSource', {
+    schemaDoc: 'The source for the address',
```

Comment:
It's unclear from this comment what the source is. Could you please write more detailed comment.

Reply:
Done